### PR TITLE
[connectedmachine] release stable version 2024-07-10

### PR DESCRIPTION
**Pull request by yaotongms**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
This release is targeting version 2024-07-10 of connectedmachine and using aaz.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedmachine

File: src/connectedmachine/setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '1.0.0b2'
+VERSION = '1.0.0'
 try:
```
Comment (AllyW) [inline src/connectedmachine/setup.py:14]: if stable version is expected, azext.isPreview: true in azext_metadata.json should be removed

**Comment by yaotongms** [comment]
aaz: https://github.com/Azure/aaz/pull/583

**Review by necusjz — APPROVED** [review]

**Comment by jkroepke** [comment]
@yaotongms What the reason why the `az connectedmachine run-command` is removed?

**Comment by yaotongms** [comment]
> @yaotongms What the reason why the `az connectedmachine run-command` is removed?

this release is targeting a stable version API that doesn't contain runcommands since it's in preview. It will be included in future release targeting preview API.

**Comment by jkroepke** [comment]
> this release is targeting a stable version API that doesn't contain runcommands since it's in preview. It will be included in future release targeting preview API.

Feels a bit sad, since the documented command are not longer working. https://learn.microsoft.com/en-us/azure/azure-arc/servers/run-command

**Comment by Sorensiim** [comment]
image

**Pull request merged**